runtime/compiler/ilgen/ClassLookahead.cpp

```diff
@@ -1470,7 +1470,7 @@ TR_PersistentFieldInfo *TR_PersistentClassInfoForFields::find(TR::Compilation *c
          if (staticsAreSame(fieldInfo->getOwningMethodSymbol()->getResolvedMethod(), fieldInfo->getFieldSymRef()->getCPIndex(), fieldSymRef->getOwningMethod(), fieldSymRef->getCPIndex()))
             return fieldInfo->asPersistentArrayFieldInfo();
          }
-      */
+
```

Comment:
It looks like the end of this comment block was accidentally removed.  Please add it back.